Add logging of queries and response to Sqlite #27

aditya0yadav opened the pull request:
[ncoop57](https://github.com/ncoop57)
[Isaac-Flath](https://github.com/Isaac-Flath)
[cshintov](https://github.com/cshintov)

add the log flag 

accord to the issue #27

ncoop57 commented:
Hey @aditya0yadav looking good! Though could please swap out SQLite utils to use fastlite? We use it for all our SQLite projects and it makes it a lot easier to use with good default's vs directly using SQLite utils

aditya0yadav commented:
> Hey @aditya0yadav looking good! Though could please swap out SQLite utils to use fastlite? We use it for all our SQLite projects and it makes it a lot easier to use with good default's vs directly using SQLite utils

sure

aditya0yadav commented:
[ncoop57](https://github.com/ncoop57)
[Isaac-Flath](https://github.com/Isaac-Flath)
[cshintov](https://github.com/cshintov)
change is commited 
use the fastlite in place of sqlite_utils

ncoop57 commented:
Sweet @aditya0yadav when I'm back at my comp in the morning I'll test it out locally. So far, it looks good!

aditya0yadav commented:
Ok thanks 

Problem is 
Default setting directory and file is not available in Gitpod when the repo is forked and open it on Gitpod

But the default setting is available when I Gitpod directly on real repo not forked one

ncoop57 commented:
@aditya0yadav could you give me edit access for the PR?

aditya0yadav commented:
![Screenshot 2025-02-08 at 10 14 10 PM](https://github.com/user-attachments/assets/818198ab-5a09-47f1-88ab-88f6c53e2af6)
i am sure i already give it 
is this it 

write access is something else

ncoop57 commented:
Hmmm, can you give me contributor access to the repo? @aditya0yadav I'm having issues making any changes :(

ncoop57 commented:
okay I'm able to make changes now. Sorry about that

ncoop57 commented:
@aditya0yadav Great job on this PR! I did some slight improvements, specifically:

1. Used a log class to make creation and insertion of logs a bit cleaner.
2. Renamed the migrate function to something more descriptive
3. Added a log option to the config so users can turn it on permanently if they wish


The only thing left is to update the index.ipynb to have the documentation of this new awesome feature and run `nbdev_readme` so the readme gets updated 🤓 Could you add that please?

aditya0yadav commented:
> @aditya0yadav Great job on this PR! I did some slight improvements, specifically:
> 
> 1. Used a log class to make creation and insertion of logs a bit cleaner.
> 2. Renamed the migrate function to something more descriptive
> 3. Added a log option to the config so users can turn it on permanently if they wish
> 
> The only thing left is to update the index.ipynb to have the documentation of this new awesome feature and run `nbdev_readme` so the readme gets updated 🤓 Could you add that please?

[ncoop57](https://github.com/ncoop57)   sure

aditya0yadav commented:
hi [ncoop57](https://github.com/ncoop57) 
docs is updated please check it out

ncoop57 commented:
Awesome!!! Thanks @aditya0yadav for this PR!
